components/blocks/code.module.css

```diff
@@ -85,6 +89,10 @@
   @apply text-yellow-50;
 }
 
+:global(.dark) .Container pre code :global(.decorator.annotation.punctuation) {
+  @apply text-yellow-50;
```

Comment:
Can we use `text-yellow-80` on both light and dark modes? To prevent the color change when switching between them:

https://user-images.githubusercontent.com/34423371/190678841-0b1275be-1076-4a74-8056-2239a3ee19a5.mov

Reply:
Fixed on 032a058